[docs] Update Zoom link creation process (#3580)
open-telemetry/community: docs/how-to-handle-public-calendar.md

You can see which Zoom account any potentially conflicting meetings are using in the meeting descriptions.
(Note: posting the URLs publicly on GitHub leads to Zoom bombing by random bots).

#### LFPCC Zoom meeting configuration

_This is the process that the person responding to Zoom link creation issues will follow. The instructions under this heading are for Governance Committee members or Technical Committee members who have access to the Project Control Center admin console._

To create a new Zoom meeting on the Project Control Center, follow these instructions:

1. Log into https://openprofile.dev/ with your Github account
2. Click on the top right on Tools and click on "Project Control Center"
3. On the left-hand column, click on "Collaboration", then "Meetings", then "Add Meeting"
4. Fill in the meeting details and make sure to select the following options:

>     - Select "Add to public calendar" if it makes sense

Reviewer: ```suggestion
    - Select "Show in Public Calendar" if it makes sense
```
